File: project/5.02_簡単なクラスの作成/py/py_ex5_02_2/src/ex5_02_2.py
'''
Created on 2023/04/29

@author: Keroichi.T
'''

class Sphere:
    radius = 0.0
    cp = []


def main():
    cube = Sphere()
    cube.radius = 5.5
    cube.cp = [ 1.2, 3.4 ]    
    
    print( "半径：{}".format( cube.radius ) )
    print( "中心座標：{},{}".format( cube.cp[0], cube.cp[1] ) )

if __name__ == '__main__':
    main()

File: project/5.02_簡単なクラスの作成/java/java_ex5_02_2/src/java_ex5_02_2/Ex5_02_2.java
package java_ex5_02_2;

class Sphere {
	double radius;					// 半径
//	double[] cp = new double[2];	// 中心座標
	double[] cp;					// 中心座標
}

public class Ex5_02_2 {

	public static void main(String[] args) {
		
		Sphere cube = new Sphere();
		cube.radius = 5.5;
		cube.cp = new double[] {1.2,3.4};

		System.out.println( "半径：" + cube.radius );
		System.out.println( "中心座標：" + cube.cp[0] + "," +  cube.cp[1] );
		
	}

}
